src/contexts/DocumentListContext.jsx

```diff
@@ -50,17 +53,17 @@ export const DocumentListContextProvider = ({ children }) => {
       setDocumentListObject({
         docList: [],
         dataset: createSolidDataset(),
-        containerUrl: `${selectedUser.podUrl}PASS/Documents/`
+        containerUrl: client ? `${client?.podUrl}PASS/Documents/` : `${podUrl}PASS/Documents/`
```

Comment:
are these correct?  Is there no `/` between the `podUrl` and `PASS`?

Reply:
Yes, since we usually have the trailing "/" as part of the `podUrl`, we don't need the extra "/" between `podUrl` and `PASS` for the container URL